lib/content-types/only.js

```diff
@@ -3,11 +3,19 @@
 const merge = require('deepmerge');
 const utils = require('../utils');
 
-const only = (type, configuration) => {
+/**
+ * Retrieves one content type's config object, merged with input plugins and a form's values
+ *
+ * @param  {object} type merged content type object
+ * @param  {object} vals values for inputs
+ *
+ * @returns {object}  content type object merged with input plugins and values
+ */
+const only = (type, vals) => {
```

Comment:
And in previous function this was called `userConfig`, and I'd still like it to be something like `overrides`